### ISSUE TITLE
Error when using TCK files written by dipy 

### ISSUE THREAD

**Issue by mick-d** [nipy/dipy#1722]
## Description
When using latest version of dipy I get the following error from any MRtrix tools when using a TCK file saved from dipy with `save_tck`. This is well illustrated with the `tckinfo` debug tool :
```
> tckinfo -debug mybundle_dipy.tck
tckinfo: [DEBUG] No config file found at "/etc/mrtrix.conf"
tckinfo: [DEBUG] No config file found at "/home/me/.mrtrix.conf"
tckinfo: [DEBUG] reading key/value file "mybundle_dipy.tck"...
tckinfo: [ERROR] invalid first line for key/value file "mybundle_dipy.tck" (expected "mrtrix tracks")
```
However this does not happen when using [another package](https://github.com/MIC-DKFZ/TractSeg/blob/06defd23775a21d3501c079ce67c686de9810c45/tractseg/libs/fiber_utils.py#L126) based on `nibabel` (cf code below).
```
> tckinfo -debug mybundle_other.tck
tckinfo: [DEBUG] No config file found at "/etc/mrtrix.conf"
tckinfo: [DEBUG] No config file found at "/home/me/.mrtrix.conf"
tckinfo: [DEBUG] reading key/value file "mybundle_other.tck"...
***********************************
  Tracks file: "mybundle_other.tck"
    count:                0000005984
    dimensions:           [145 174 145]
    voxel_order:          b'RAS'
    voxel_sizes:          [ 1.25  1.25  1.25]
```

### Code to reproduce error
```
import numpy as np
from dipy.tracking.streamline import Streamlines
from dipy.io.streamline import load_trk, save_tck # with dipy: does not work (causes MRtrix error)
from tractseg.libs import fiber_utils # with other package: works (does not cause MRtrix error)

# Read trk file
streams, hdr = load_trk('mybundle.trk')
streamlines = Streamlines(streams)

# Convert to tck with dipy: does not work
save_tck("mybundle_dipy.tck", streamlines, affine=hdr['voxel_to_rasmm'], 
         vox_size=None, shape=hdr['dimensions'], header=hdr)

# Convert to tck with other package: works
fiber_utils.save_streamlines("mybundle_other.tck", streamlines, affine=hdr['voxel_to_rasmm'], 
                             shape=hdr['dimensions'], vox_sizes=hdr['voxel_sizes'], 
                             vox_order=hdr['voxel_order'])

```
Please note that the other package used to lead to the same error, and this was fixed (in the context of the other package) with the [following commit](https://github.com/MIC-DKFZ/TractSeg/commit/06defd23775a21d3501c079ce67c686de9810c45).

**Comment by skoudoro** [comment]
Hi @mick-d, Thank you for this full report.  Can you tell me if it works when you setup `vox_size` with `hdr['voxel_sizes']`like you did with the other package ?

but we definitively have to update this point instead of being silent

**Comment by Garyfallidis** [comment]
Could it be that the tck version has changed?

**Comment by mick-d** [comment]
I do not think so, as per [this post](http://community.mrtrix.org/t/error-invalid-first-line-for-key-value-file/2177) from a few days ago:

> "I don’t think there’s been any significant changes to the track file writing code for some time."

**Comment by skoudoro** [comment]
fixed by #1812, closing
